docs-app/.template-lintrc.js

```diff
@@ -4,6 +4,12 @@ module.exports = {
   plugins: ['ember-template-lint-plugin-prettier'],
   extends: ['recommended', 'ember-template-lint-plugin-prettier:recommended'],
   overrides: [
+    {
+      files: ['**/*.{gjs,gts}'],
+      rules: {
+        'no-implicit-this': 'off',
+      },
+    },
```

Comment:
`ember-template-lint` support for `*.{gjs,gts}` files isn't great at the moment. For simplicity, we won't enforce `no-implicit-this` for now.